src/log_batch.rs

```diff
@@ -896,6 +929,31 @@ fn verify_checksum(buf: &[u8]) -> Result<()> {
     Ok(())
 }
 
+/// Verifies the checksum of a slice of bytes that sequentially holds data and
+/// checksum with the given `[signature]`.
```

Comment:
Again, [xxx] is to link an external name. Here signature is not a global name.
```suggestion
/// checksum generated with the given `file_context`.
```